time timeStamp
int32 recvCarId
int32 srcCarId
float32 lat
float32 lon
float32 vel
float32 front_distance
bool is_signal_on
int32 signal_direction
int32 dilema_id
bool is_source_car_safe
bool is_dest_car_safe
float32 tof0
float32 tof1
float32 tof2
float32 tof3
float32 tof4
float32 tof5